src/plugins/intel_cpu/src/nodes/fullyconnected.h

```diff
@@ -100,13 +104,34 @@ class FullyConnected : public Node {
     float weiSparseRate = 0.f;
     bool useSparseWeightsDecompression();
     VectorDims expectedBiasDims {};
+
     bool useMlas = false;
 #ifdef OV_CPU_WITH_MLAS
     int64_t M, N, K;
     MemoryPtr mlasPackedPtr = nullptr;
     void executeMLAS();
     void prepackMLASWeight();
 #endif
+
+#ifdef OV_CPU_WITH_LLMDNN
+    bool tryExtractParamForLLMFc(llmdnn::fc_create_param& param);
+    bool tryUseLLMFc();
```

Comment:
Could you change the function name to make it more accurate ? The function name better implies its functionality. For example,

1.  `tryUseLLMFc` is used to initalize the kernel and prepack the weight, is `initLLMFc` better ?
2.  `tryExecLLMFc` is used to execute the kernel, could you simplify it to execLLM ?

Reply:
Fixed.